charts/jaeger/templates/_helpers.tpl

```diff
@@ -719,3 +719,95 @@ Create pull secrets for hotrod image
 {{- define "hotrod.imagePullSecrets" -}}
 {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.hotrod.image) "context" $) -}}
 {{- end }}
+
+
+
+{{- define "jaeger-v2.name" -}}
```

Comment:
Why any of these changes below necessary? Doesn't the chart already have them all for v1? Can we not reuse the same variables?

Reply:
yes that is what i planed to do once i was sure that i was going in the right direction with v-2 deployment and config part